dali/python/nvidia/dali/ops.py

```diff
@@ -536,15 +538,23 @@ def _repack_list(self, sets, fn):
 
 
     Operator.__name__ = str(name)
+    Operator.schema_name = schema_name or Operator.__name__
     # The autodoc doesn't generate doc for something that doesn't match the module name
-    if _b.GetSchema(Operator.__name__).IsInternal():
+    if _b.GetSchema(Operator.schema_name).IsInternal():
         Operator.__module__ = Operator.__module__ + ".internal"
 
-    Operator.__call__.__doc__ = _docstring_generator_call(name)
+    Operator.__call__.__doc__ = _docstring_generator_call(Operator.schema_name)
     return Operator
 
-def _wrap_op(op_class):
-    return _functional._wrap_op(op_class)
+def _process_op_name(op_full_name):
+    if not op_full_name.startswith('_'):
```

Comment:
When using two underscores, this condition should not be necessary. In fact, we may want to hide an entire module this way.